documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -238,6 +238,39 @@ If you generate an optimization proposal using the default optimization goals, C
 To change the cached optimization proposal refresh interval, edit the `proposal.expiration.ms` setting in the Cruise Control deployment configuration.
 Consider a shorter interval for fast changing clusters, although this increases the load on the Cruise Control server.
 
+[id='con-optimization-proposals-modes-{context}']
+== Rebalancing modes with broker scaling
```

Comment:
I agree this is probably a better place than in the tuning options and maybe for now we can leave it here. But I do think that we need to do a bit of an overhaul of the CC docs to introduce the modes nearer the top with a more comprehensive introduction. I will open an issue to cover that.

Reply:
@tomncooper out of curiosity ... what do you think we should add to have "a more comprehensive introduction". Isn't the current one covering everything? Should we already improve it?
Other question is about the title here ... I don't think that "with broker scaling" part is right, because when talking about "full" mode, it doesn't involve scaling.